lib/modules/dosomething/dosomething_campaign/dosomething_campaign.admin.inc

```diff
@@ -499,3 +502,56 @@ function dosomething_campaign_archive_activity_form_submit(&$form, &$form_state)
     '@target' => $target_nid,
   )));
 }
+
+/*
+ * Organ donation configuration form. Allows admins to turn on/off the
+ * organ donation modal after signup.
+ *
+ */
+function dosomething_campaign_organ_donation_config_form($form, &$form_state, $node) {
```

Comment:
Now that i'm looking at it, i feel like this admin config stuff should be housed in the `dosomething_organ_donation` module. But it's not a blocker for merge, I don't think...